python/tvm/script/builder/tir/prim_func_frame.py

```diff
@@ -40,3 +40,71 @@ def arg(name, obj) -> Union[Var, Buffer]:
 
 
 setattr(prim_func, "dispatch_token", "tir")
+
+
+def func_attr(attrs: Dict[str, Any]) -> None:
+    return _ffi_api.FuncAttrs(attrs)  # pylint: disable=no-member # type: ignore
+
+
+def func_ret(ret_type) -> Union[Var, Buffer]:
```

Comment:
The return type is probably wrong (?) i guess it's something like

```suggestion
from tvm.ir import Type

def func_ret(ret_type) -> Type:
```